strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/layout/AbstractLayoutProvider.java

```diff
@@ -103,35 +88,6 @@ public Storage getStorage(String storageId)
         return configurationManager.getConfiguration().getStorage(storageId);
     }
 
-    @Override
```

Comment:
I understand you have removed both the `getInputStream` and `getOutputStream` methods. I don't think I fully understand the reasoning behind this, or that I agree/approve.

I think that the layout provider (whether the abstract implementation, or the actual implementation) should provide these methods for convenience. They can be doing the `Files.newInputStream` there, or whatever the logic should be. Right now we're talking about file-system like `InputStream`-s. What will we do in the future, if, for example, we decide to have a database provider (or something along these lines) where `Files.newInputStream` will probably not be applicable? A non-Strongbox-core developer/user of our code should be able to figure out how to do things with the least amount of effort and I think removing these methods that hide the underlying implementation will lead to some confusion in the future, as people will be wondering how to get an input/output stream and will end up doing their own thing, no necessarily being `Files.get[Input|Output]Stream`.

What are your thoughts on this?